en/ui-system/components/editor/richtext.md

```diff
@@ -63,14 +86,17 @@ is equal to:
 
 `<color=green><size=30>I'm green</size></color>`
 
-<!--
-There are two ways to set the color of RichText:
-1. Selected the node and set the overall color of RichText in __RichTextComponent -> String__ of the __Inspector__.
-2. Use BBCode to set colors on the inside of RichText separately.
-Use BBCode to set the colors separately inside RichText.
+## Cache Mode
+
+Since the RichText component is assembled from multiple Label nodes, the number of drawcalls for complex rich text will also be high. Therefore, the engine provides the CacheMode setting of the Label component for the RichText component to avoid the increase of drawcall. For a detailed description of each cache type, refer to [Cache Mode of the Label component](./label.md)
```

Comment:
For a detailed description of each cache type, refer to [Cache Mode of the Label component](./label.md)
-->
For a detailed description of each cache type, please refer to the [Cache Mode of the Label component](./label.md) documentation.